### DISCUSSION TITLE
rook-ceph Device type is not acceptable

### DISCUSSION THREAD

**Discussion by slehlib** [rook/rook#8893] [General]
Hello,
kubernetes: 1.22.2
Rook-ceph: 1.7.3
Os : Redhat 7.9

I want to deploy a new ceph cluster with rook in bare metal server with multipathd but i cannot create the osd pod 
in the log i have this message:
exec: Running command: ceph-volume inventory --format json /dev/mapper/GK5CLI_1c00_CEPH01
cephosd: skipping device "dm-2": ["Device type is not acceptable. It should be raw device or partition"].exec: Running command: ceph-volume inventory --format json /dev/mapper/GK5CLI_1c00_CEPH01
cephosd: skipping device "dm-2": ["Device type is not acceptable. It should be raw device or partition"].
Ceph support multipathd ? any solution to fix it 
Thanks

**Comment by leseb** [comment]
We've had enabled that in the past in Rook, but it looks like recent changes in `ceph-volume` might break this. I suggest chatting with the developers directly by opening an issue here https://tracker.ceph.com/. Thanks!

**Comment by slehlib** [comment]
thanks for help.
can you give me the rook version work with it ?

**Reply by slehlib** [reply]
@leseb any news about this ?

**Reply by leseb** [reply]
Where is the ceph tracker? Did you open it?

**Reply by slehlib** [reply]
this is tracker
https://tracker.ceph.com/issues/52908
